support/cas-server-support-consent-webflow/build.gradle

```diff
@@ -8,4 +11,5 @@ dependencies {
     implementation project(":support:cas-server-support-consent-api")
     implementation project(":core:cas-server-core-configuration")
     implementation project(":core:cas-server-core-authentication")
+    implementation project(":webapp:cas-server-webapp-config")
```

Comment:
This cannot happen. webapp modules may not be accessible from support modules. I recommend you move the new interceptor into a more commonly-shared module. How about something in pac4j-core, etc?

Reply:
This should be a core module then, right? Would it fit in cas-server-core-web (org.apereo.cas.web.pac4j)? Else I'd suggest a new core module cas-server-core-pac4j.